patch.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This was added by accident, I guess?

Reply:
Ah yes, I need this patch to build the lib in vcpkg